.idea/inspectionProfiles/Project_Default.xml

```diff
@@ -1,6 +1,9 @@
 <component name="InspectionProjectProfileManager">
   <profile version="1.0">
     <option name="myName" value="Project Default" />
+    <inspection_tool class="ExtractMethodRecommender" enabled="true" level="WARNING" enabled_by_default="true">
```

Comment:
Can you revert this?

If this is for some reason useful please extract to a dedicated PR